.github/workflows/ci.yml

```diff
@@ -29,8 +29,13 @@ jobs:
     - if: matrix.runs-on == 'windows-latest' && github.ref != 'refs/heads/main' && !contains(github.ref, 'refs/tags/')
       run: ./build.ps1
     - if: (matrix.runs-on == 'windows-latest') && (github.ref == 'refs/heads/main' || contains(github.ref, 'refs/tags/'))
-      env:
-        SignClientSecret: ${{ secrets.SIGNCLIENT }}
       run: |
-        ./build.ps1 sign
-        dotnet nuget push .\nuget\*.nupkg -s https://www.myget.org/F/identity/api/v2/package -k ${{ secrets.MYGET }}
+        ./build.ps1
+        # dotnet nuget push .\nuget\*.nupkg -s ${{ secrets.NUGET_REPO_URL }} -k ${{ secrets.NUGET_REPO_API_KEY }}
```

Comment:
Can you please add a comment on why this `dotnet nuget push` line has been changed and at the same time also commented out? Maybe a short note about how Nugets are being built with the new way. Thanks! :-)

Reply:
The fact that the nugets are not automatically uploaded to nexus is already mentioned on the top of readme.md. But if you want me to I can also mention it here in the ci.yml directly...

Reply:
Thanks for cleaning it up in the end! This way we can avoid any confusion of future readers 👍 😉